dll/win32/CMakeLists.txt

```diff
@@ -59,6 +59,7 @@ add_subdirectory(ieframe)
 add_subdirectory(iernonce)
 add_subdirectory(imaadp32.acm)
 add_subdirectory(imagehlp)
+#add_subdirectory(imageres)
```

Comment:
@learn-more I'd initially planned to use `imageres` to store the Show Desktop button's icon (thus following Windows's example), and had matched the resource...identifier??? with Windows and everything.

However, another copy of the same icon is already present in `shell32`. As such, doing this would be violating DRY (as Windows does here...for some reason...), and thus I was encouraged to not add `imageres` to ROS, and just use the icon from `shell32` instead. Thus, I did.

That said, I thought I'd already removed all traces of/references to `imageres` from this branch...yet your review indicates otherwise? Is the stuff still there after all?